File: 16-security/npm-package/security.ts
import axios from 'axios';
const API_BASE = 'https://cost-katana-backend.store/api'\;
const API_KEY = process.env.COST_KATANA_API_KEY;

async function securityScan(type: string, data: any) {
  const response = await axios.post(
    `${API_BASE}/security/scan`,
    { type, data },
    { headers: { 'Authorization': `Bearer ${API_KEY}` } }
  );
  console.log('Security scan:', response.data);
  return response.data;
}

export { securityScan };

File: 16-security/python-sdk/security.py
import os, requests
API_BASE = "https://api.costkatana.com/api"
API_KEY = os.getenv("COST_KATANA_API_KEY")

def security_scan(scan_type, data):
    response = requests.post(
        f"{API_BASE}/security/scan",
        json={"type": scan_type, "data": data},
        headers={"Authorization": f"Bearer {API_KEY}"}
    )
    response.raise_for_status()
    print("Security scan:", response.json())
    return response.json()
